Main.py

```diff
@@ -314,7 +320,7 @@ def precollect_hint(location: Location, auto_status: HintStatus):
                     if current_sphere:
                         spheres.append(dict(current_sphere))
 
-                multidata = {
+                multidata: NetUtils.MultiData | bytes = {
```

Comment:
I don't love using a union as a sort of workaround for an error from assigning a different type to a variable. Do you think it'd be better to rename one of them instead?

Reply:
I would also separate it into two variables, but that's better done as a followup PR.